src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_LENGTH_OF_SIDE = 10;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor().toString();
+        int index = new Random().nextInt(Figures.values().length);
+        Figures figure = Figures.values()[index];
```

Comment:
let's remove `enum Figures` and choose figure in switch case depending on random number from 0 to 4, where 0 must return random square for example and so on ...